clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -49,8 +62,19 @@
   }
 )
 
+(define-map registered-swap-tokens
+  { swap-token: principal }
+  { registered: bool }
+)
+
 (define-data-var pair-count uint u0)
-(define-data-var pairs-list (list 2000 uint) (list ))
+
+(define-read-only (is-shutdown-activated)
+  (and
+     (not (unwrap-panic (contract-call? .arkadiko-dao get-emergency-shutdown-activated)))
+     (not (var-get swap-shutdown-activated))
+  )
```

Comment:
The boolean logic in this function is a bit counter intuitive / name of the function is misleading.
The result or your operation is indicating that shutdown is **not** activated.

Reply:
Agree - I have renamed the method.